static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -70,107 +67,90 @@ function GroupRelatedIssues({params}: Props) {
   );
 
   return (
-    <Layout.Body>
-      <Layout.Main fullWidth>
```

Comment:
The Layout component is now at the top of the Similar Issues component wrapping both sections.
<img width="480" alt="image" src="https://github.com/getsentry/sentry/assets/44410/0fd62ed4-2e50-469c-aa31-ea53e9826e0f">